trajectory_msgs/JointTrajectory   points
---
bool                              valid
int32[]                           invalid_indices